### DISCUSSION TITLE
What is the default Admin Side Panel Search Behavior?

### DISCUSSION THREAD

**Discussion by pakman6392** [wagtail/wagtail#11624] [General]
Hello,
My team recently noticed an issue with our search bar within the Admin sidebar and we are unsure if it should be considered a bug or an issue that we introduced ourselves into the code. 

We are currently running Django 4.2, Wagtail 5.2, and Python 3.11. We are also using the elastcsearch7 backend if this information is helpful.

The issue is that the search is being treated as an AutocompleteFiled search. Normally, all of our users type something into the search bar and then search the results. However, we did have one person who just hit the search icon which caused an error to be thrown and then letting us developers know about it. I believe it was an issue of an empty query_string value, but I don't remember for sure off the top of my head. We put a fix in place for us for now by informing our users of the issue as a workaround, but I wanted to make sure that this was not the standard behavior before creating an issue. We also are running our own search backend which is why I wanted to check with you guys regarding the default behavior.

Thanks

**Comment by gasman** [comment]
Thanks for the report! There's an existing issue about this at #11600 (and a fix in progress at #11601).

**Comment by pakman6392** [comment]
Thank you for letting me know.
